app/globals.css

```
@@ -373,6 +374,24 @@
     color: var(--mundia-paper);
   }
 
+  .catalog-page-link {
+    @apply inline-flex min-h-12 items-center justify-center rounded-lg border text-sm font-semibold transition-colors;
+    border-color: var(--mundia-line);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)